spec/support/coronavirus_feature_steps.rb

```diff
@@ -450,12 +450,12 @@ def stub_discard_coronavirus_page_no_draft
   end
 
   def i_see_subsection_one_in_position_one
-    element = find("#step-0")
+    element = find(".app-c-reorderable-list__item:first-child")
     expect(element).to have_content "I am first"
```

Comment:
I think we can avoid peering into the component too much here by doing:

 expect(find(".app-c-reorderable-list")).to have_content /^I am first/